app/models/user.rb

```diff
@@ -35,7 +35,7 @@ class User < ApplicationRecord
   # email presence validated through Devise
   validates :name, :role, presence: true
   validates_format_of :email, with: Devise::email_regexp
-  validate :secure_password
```

Comment:
This is already the case on `main`, so I'm a bit sus of this diff? do you need to pull in `main`?

 https://github.com/DCAFEngineering/dcaf_case_management/blob/c5811041293e634bbdcb560311a9c464e3b749d1/app/models/user.rb#L85

Reply:
super plausible there's some rebasing out of sync. I'll check on this.

Reply:
This is definitely a case of me accidentally screwing up a rebase conflict. See https://github.com/DCAFEngineering/dcaf_case_management/pull/2143/commits/58d30ecb15404c74603964054d6c6007b4e66881 for my exact mistake on the pg-shell branch. I think it's probably fine to leave this diff as atonement for my prior error but interested in your opinion.

Reply:
👍🏼 I don't spy any others.